### PR TITLE
Version bump 2.0.0

### PR DESCRIPTION
**Pull request by ahal**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+Contributing to Simple Github
+=============================
+
+Thanks for your interest in simple-github! To participate in this community, please
+review our [code of conduct][0].
+
+[0]: https://github.com/taskcluster/taskgraph/blob/main/CODE_OF_CONDUCT.md
```
Comment (jcristau) [inline CONTRIBUTING.md:7]: https://github.com/mozilla-releng/simple-github/blob/main/CODE_OF_CONDUCT.md

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+Clone the Repo
+--------------
+
+To contribute to simple-github, you'll need to clone the repository:
+
+```
+# first fork taskgraph
```
Comment (jcristau) [inline CONTRIBUTING.md:15]: simple-github

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+git clone https://github.com/<user>/simple-github
+cd simple-github
+git remote add upstream https://github.com/mozilla-releng/simple-github
+```
+
+Setting Up the Environment
+--------------------------
+
+We use a tool called [uv][1] to manage Taskgraph and its dependencies. First,
```
Comment (jcristau) [inline CONTRIBUTING.md:24]: s/taskgraph/simple-github/

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+```
+pre-commit run
+```
+
+Most of the checks we enforce are done with [ruff][5]. See
+[pre-commit-config.yaml][6] for a full list of linters and formatters.
+
+[4]: https://pre-commit.com/
+[5]: https://docs.astral.sh/ruff/
+[6]: https://github.com/taskcluster/taskgraph/blob/main/.pre-commit-config.yaml
```
Comment (jcristau) [inline CONTRIBUTING.md:77]: s/taskgraph/simple-github/

**Comment by ahal** [comment]
Lol.. I guess you can guess where I copied this file from

**Review by jcristau — APPROVED** [review]

**Pull request merged**